grouper/graph.py

```diff
@@ -42,15 +46,10 @@ def Graph():
     return GroupGraph()
 
 
-# A GroupGraph caches users, permissions, and groups as objects which are intended
-# to behave like the corresponding models but without any connection to SQL
-# backend.
-PermissionTuple = namedtuple(
```

Comment:
Looks like as more cleanup it should be possible to remove the current `permissions` field (seems to be unused) and rename `permission_tuples` -> `permissions` (+ `_get_permission_tuples` -> `get_permissions`). Think that'd clarify things a fair bit.

Reply:
Good idea for further cleanup. `permissions` is sadly used in the API code directly, but I think there's a lot of opportunity here to clean this up, and I'll queue that up for a later diff.